### ISSUE TITLE
LibYAML error

### ISSUE THREAD

**Issue by tinyhare** [teepark/reconfig#1]
I used this package, and get this 

> Traceback (most recent call last):
> File "./test.py", line 11, in <module>
>   cof.load()
>  File "/root/yaoqi/preload/venv/lib/python2.7/site-packages/reconfig.py", line 23, in load
>    newconfig = yaml.load(raw, Loader=yaml.CSafeLoader)
> AttributeError: 'module' object has no attribute 'CSafeLoader'

I think it's because I didn't use LibYAML for PyYAML ,so I think it should support this situation
I add these code:

```
try:
    from yaml import CSafeLoader as SafeLoader
except ImportError:
    from yaml import SafeLoader
```

and change 
`newconfig = yaml.load(raw, Loader=yaml.CSafeLoader)` 
to 
`newconfig = yaml.load(raw, Loader=SafeLoader)`

**Comment by teepark** [comment]
Can you submit that as a pull request?

**Comment by teepark** [comment]
addressed in #2
